mpds_aiida/calc_templates/fleur_default.yml

```diff
@@ -1,6 +1,6 @@
 codes:
   fleur: fleur@yascheduler
-  inpgen: inpgen@yascheduler
+  inpgen: inpgen@local_machine
```

Comment:
@blokhin I do not see any reason to make inpgen work via yascheduler, or am I wrong here?